### PR TITLE
Update Documents/<part_num>.html to reflect current set of examples supported.

### PR DESCRIPTION
**Pull request by lorne-maxim**

### CODE REVIEW

File: Documentation/MAX32655.html [hunk 1]
```diff
@@ -55,27 +55,41 @@ <h2>Example Applications</h2>
...
 			<li><a href='../Examples/MAX32655/I2C/README.md'>I2C</a> - The Standard 2-Wire Serial Protocol</li>
+			<li><a href='../Examples/MAX32655/I2C_MNGR/README.md'>I2C_MNGR</a> - Demonstration of managing simultaneous I2C transaction requests</li>
+			<li><a href='../Examples/MAX32655/I2C_SCAN/README.md'>I2C_SCAN</a> - Demonstration of discovering I2C devices on the bus</li>
+			<li><a href='../Examples/MAX32655/I2C_Sensor/README.md'>I2C_Sensor</a> - Demonstration of communicating with an I2C temperature sensor</li>
 			<li><a href='../Examples/MAX32655/I2S/README.md'>I2S</a> - Inter-IC Sound</li>
+			<li><a href='../Examples/MAX32655/I2S_Playback/README.md'>I2S_Playback</a> - Demonstration of playback of audio sample using I2S</li>
 			<li><a href='../Examples/MAX32655/ICC/README.md'>ICC</a> - Internal Cache Controller</li>
 			<li><a href='../Examples/MAX32655/LP/README.md'>LP</a> - Low Power</li>
+			<li><a href='../Examples/MAX32655/LPCMP/README.md'>LPCMP</a> - Low Power Comparator</li>
 			<li><a href='../Examples/MAX32655/Pulse_Train/README.md'>Pulse_Train</a></li>
```
Comment (Jacob-Scheiffler) [inline Documentation/MAX32655.html:90]: I believe RF_Test is missing here.

File: Documentation/MAX32672.html [hunk 2]
```diff
@@ -55,20 +55,27 @@ <h2>Example Applications</h2>
 		<ul>
 			<li><a href='../Examples/MAX32672/ADC/README.md'>ADC</a> - Analog-to-Digital Converter</li>
 			<li><a href='../Examples/MAX32672/AES/README.md'>AES</a> - Advanced Encryption Standard</li>
+			<li><a href='../Examples/MAX32672/ARM-DSP/README.md'>DSP</a> - Digital Signal Processing</li>
 			<li><a href='../Examples/MAX32672/Comparator/README.md'>Comparator</a></li>
 			<li><a href='../Examples/MAX32672/CRC/README.md'>CRC</a> - Cyclic Redundancy Check</li>
 			<li><a href='../Examples/MAX32672/CTB_AES/README.md'>CTB_AES</a> - Cryptographic Toolbox Advanced Encryption Standard</li>
 			<li><a href='../Examples/MAX32672/DMA/README.md'>DMA</a> - Direct Memory Access</li>
```
Comment (Jacob-Scheiffler) [inline Documentation/MAX32672.html:62]: Missing Display example.

File: Documentation/MAX32690.html [hunk 3]
```diff
@@ -76,7 +77,9 @@ <h2>Example Applications</h2>
 			<li><a href='../Examples/MAX32690/GPIO/README.md'>GPIO</a> - General Purpose Input/Output</li>
 			<li><a href='../Examples/MAX32690/Hello_World/README.md'>Hello_World</a></li>
```
Comment (Jacob-Scheiffler) [inline Documentation/MAX32690.html:78]: Hash and HBMC examples missing.

File: Documentation/MAX78002.html [hunk 4]
```diff
@@ -63,48 +63,51 @@ <h2>Example Applications</h2>
...
-				<li><a href='../Examples/MAX78002/CNN/mnist-riscv/README.md'>mnist-riscv</a> - Handwritten Digit Recognition (runs on RISC-V core)</li>
-				<li><a href='../Examples/MAX78002/CNN/snake_game_demo/README.md'>snake_game_demo</a> - Audio Keyword Spotting</li>
+				<li><a href='../Examples/MAX78002/CNN/cifar-100-effnet2/README.md'>cifar-100-effnet2</a> - Object Classification using 100 Choices</li>
+				<li><a href='../Examples/MAX78002/CNN/cifar-100-mobilenet-v2-0.75/README.md'>cifar-100-mobilenet-v2-0.75</a> - Object Classification using 100 Choices</li>
+				<li><a href='../Examples/MAX78002/CNN/imagenet/README.md'>imagenet</a></li>
 			</ul>
 			<li><a href='../Examples/MAX78002/CRC/README.md'>CRC</a> - Cyclic Redundancy Check</li>
 			<li><a href='../Examples/MAX78002/CSI2/README.md'>CSI2</a> - MIPI CSI-2 D-PHY</li>
+			<li><a href='../Examples/MAX78002/CameraIF/README.md'>CameraIF</a> - Parallel Camera Interface</li>
 			<li><a href='../Examples/MAX78002/DMA/README.md'>DMA</a> - Direct Memory Access</li>
```
Comment (Jacob-Scheiffler) [inline Documentation/MAX78002.html:76]: Missing CameraIF_Debayer

File: Documentation/MAX78002.html [hunk 5]
```diff
@@ -63,48 +63,51 @@ <h2>Example Applications</h2>
...
-			<li><a href='../Examples/MAX78002/I2C/README.md'>I2C_SCAN</a> - Scans for Connected Minion Devices</li>
+			<li><a href='../Examples/MAX78002/I2C_MNGR/README.md'>I2C_MNGR</a> - Demonstration of managing simultaneous I2C transaction requests</li>
+			<li><a href='../Examples/MAX78002/I2C_SCAN/README.md'>I2C_SCAN</a> - Demonstration of discovering I2C devices on the bus</li>
+			<li><a href='../Examples/MAX78002/I2C_Sensor/README.md'>I2C_Sensor</a> - Demonstration of communicating with an I2C temperature sensor</li>
 			<li><a href='../Examples/MAX78002/I2S/README.md'>I2S</a> - Inter-IC Sound</li>
-			<li><a href='../Examples/MAX78002/I2S/README.md'>I2S_DMA</a> - Inter-IC Sound DMA</li>
+			<li><a href='../Examples/MAX78002/I2S_DMA/README.md'>I2S_DMA</a> - Inter-IC Sound DMA</li>
 			<li><a href='../Examples/MAX78002/ICC/README.md'>ICC</a> - Internal Cache Controller</li>
 			<li><a href='../Examples/MAX78002/LP/README.md'>LP</a> - Low Power</li>
 			<li><a href='../Examples/MAX78002/LPCMP/README.md'>LPCMP</a> - Low Power Comparator</li>
```
Comment (Jacob-Scheiffler) [inline Documentation/MAX78002.html:91]: Missing ImgCapture

**Review by Jacob-Scheiffler — CHANGES_REQUESTED** [review]

File: Documentation/MAX32655.html (hunk 1 above)
Comment (lorne-maxim) [reply Documentation/MAX32655.html:90]: This was intentional.  There is a BLE_fcc that seems to being the same thing.  I need Kevin to weigh in on if both or needed.

File: Documentation/MAX32672.html (hunk 2 above)
Comment (lorne-maxim) [reply Documentation/MAX32672.html:62]: Correct.  It needs a README.md file to link to.  I have another task on the TODO list to clean up the few examples that are missing the readme.

File: Documentation/MAX32690.html (hunk 3 above)
Comment (lorne-maxim) [reply Documentation/MAX32690.html:78]: Adding them in now.

File: Documentation/MAX78002.html (hunk 4 above)
Comment (lorne-maxim) [reply Documentation/MAX78002.html:76]: Added.

File: Documentation/MAX78002.html (hunk 5 above)
Comment (lorne-maxim) [reply Documentation/MAX78002.html:91]: Added.

File: Documentation/MAX32680.html [hunk 6]
```diff
@@ -56,17 +56,22 @@ <h2>Example Applications</h2>
 			<li><a href='../Examples/MAX32680/ADC/README.md'>ADC</a> - Analog-to-Digital Converter</li>			
 			<li><a href='../Examples/MAX32680/AES/README.md'>AES</a> - Advanced Encryption Standard</li>
 			<li><a href='../Examples/MAX32680/AFE_ADC/README.md'>ADC</a> - Analog Front-End Analog-to-Digital Converter</li>	
+			<li><a href='../Examples/MAX32680/ARM-DSP/README.md'>DSP</a> - Digital Signal Processing</li>
 			<li><a href='../Examples/MAX32680/BLE_dats/README.md'>BLE_dats</a> - Bluetooth Data Server</li>
 			<li><a href='../Examples/MAX32680/BLE_ctr/README.md'>BLE_ctr</a> - Bluetooth v4.2 Controller</li>
```
Comment (sihyung-maxim) [inline Documentation/MAX32680.html:61]: Should be BLE4_ctr instead of BLE_ctr.

File: Documentation/MAX78000.html [hunk 7]
```diff
@@ -63,53 +63,72 @@ <h2>Example Applications</h2>
...
 				<li><a href='../Examples/MAX78000/CNN/kws20_v3/README.md'>kws20_v3</a> - Audio Keyword Spotting</li>
-				<li><a href='../Examples/MAX78000/CNN/mnist/README.md'>mnist</a> - Handwritten Digit Recognition</li>
 				<li><a href='../Examples/MAX78000/CNN/mnist-riscv/README.md'>mnist-riscv</a> - Handwritten Digit Recognition (runs on RISC-V core)</li>
 				<li><a href='../Examples/MAX78000/CNN/mnist-streaming/README.md'>mnist-streaming</a> - Handwritten Digit Recognition</li>
+				<li><a href='../Examples/MAX78000/CNN/mnist/README.md'>mnist</a> - Handwritten Digit Recognition</li>
+				<li><a href='../Examples/MAX78000/CNN/rps-demo/README.md'>rps-demo</a> - Rock, Paper, Scissors Demo</li>
 				<li><a href='../Examples/MAX78000/CNN/rps/README.md'>rps</a> - Rock, Paper, Scissors Hand Gesture Recognition</li>
-				<li><a href='../Examples/MAX78000/CNN/rps_demo/README.md'>rps_demo</a> - Rock, Paper, Scissors Demo</li>
 				<li><a href='../Examples/MAX78000/CNN/snake_game_demo/README.md'>snake_game_demo</a> - Audio Keyword Spotting</li>
-				<li><a href='../Examples/MAX78000/CNN/UNet-demo/README.md'>UNet-demo</a> - U-Net Network Demo</li>
```
Comment (sihyung-maxim) [inline Documentation/MAX78000.html:87]: Are we waiting on the README for CNN/svhn_tinierssd before including that example in the list?

Comment (lorne-maxim) [reply Documentation/MAX78000.html:87]: Yes.

File: Documentation/MAX32680.html (hunk 6 above)
Comment (lorne-maxim) [reply Documentation/MAX32680.html]: Fixed.

**Review by Jacob-Scheiffler — APPROVED** [review]

**Pull request merged**